lib/graph.rs

```diff
@@ -6,6 +6,8 @@ use differential_dataflow::collection::Collection;
 use differential_dataflow::operators::consolidate::Consolidate;
 use differential_dataflow::lattice::Lattice;
 use timely::dataflow::scopes::Scope;
+use timely::order::TotalOrder;
```

Comment:
I understand we currently don't `rustfmt` `lib/`, but given that it's Rust code I do believe that we should. Mind running the program on the newly added code?